Makefile

```diff
@@ -4,6 +4,18 @@ SMARTAPI_PATH = openapi/data_repository_service.smartapi.yaml
 SMARTAPI_PART_PATH = openapi/data_repository_service.smartapi.yaml.part
 SWAGGER2OPENAPI_PATH = swagger2openapi
 
+REPO_URL ?= "https://github.com/$(TRAVIS_REPO_SLUG)"
+BRANCH_NAME ?= $(echo "$(TRAVIS_BRANCH)" | awk '{print tolower($0)}')
+DOCS_STAGING_PATH = deploy
+
+# Make docs available at / for master and at /preview/{branch} for all others.
+# We don't need to check if `branch == gh-pages` since Travis will skip it by default.
+ifeq "$(BRANCH_NAME)" "master"
```

Comment:
For WES, we had agreed to make `develop` the main "source of truth" for docs, status badges, and GitHub Pages content. While `master` would be the default branch visible to new visitors (post 1.0), it mostly functions as a passthrough for releases.

I don't think this is worth addressing here, but making a note that we should try to sync up on branch usage, PR procedure, etc. across repos (whether it's following "HubFlow" or something else).